src/astro/utils/load.py

```diff
@@ -58,28 +66,82 @@ def load_file_into_dataframe(
         elif filetype == FileType.JSON:
             dataframe = pd.read_json(stream, **kwargs)
         elif filetype == FileType.NDJSON:
-            dataframe = pd.read_json(stream, lines=True, **kwargs)
+            if database is None:
+                raise ValueError(
+                    "database param cannot be None when the file type is Ndjson."
+                )
+            dataframe = flatten_ndjson(
+                normalize_config, stream=stream, database=database
+            )
         elif filetype == FileType.PARQUET:
             dataframe = pd.read_parquet(stream, **kwargs)
         else:
             raise ValueError(f"Unable to load file '{filepath}' of type '{filetype}'")
         return dataframe
 
 
+def flatten_ndjson(
+    normalize_config: Union[None, dict], stream: io.TextIOWrapper, database: Database
+) -> pd.DataFrame:
+
+    normalize_config = normalize_config or {}
+
+    check_ndjson_config_delimiter(database=database, normalize_config=normalize_config)
+
+    df = None
+    rows = stream.readlines(DEFAULT_CHUNK_SIZE)
+    while len(rows) > 0:
+        if df is None:
+            df = pd.DataFrame(
+                pd.json_normalize([json.loads(row) for row in rows], **normalize_config)
+            )
+        else:
+            df = df.append(
+                pd.json_normalize([json.loads(row) for row in rows], **normalize_config)
+            )
+        rows = stream.readlines(DEFAULT_CHUNK_SIZE)
+    return df
+
+
+def check_ndjson_config_delimiter(database: Database, normalize_config: dict) -> dict:
+    replacement = "_"
+    illegal_char = "."
+    if database in [Database.BIGQUERY, Database.SNOWFLAKE]:
```

Comment:
Do we want to have Astro normalizing NDJSONs in different ways, by default, depending on the database? Would it be an option to use `_` or some other character as the default delimiter, without introducing additional complexity?